Recipes/Recipes.API/Controllers/RecipesController.cs

```diff
@@ -1,307 +1,79 @@
 using System.Security.Claims;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore;
-using Recipes.DataAccess.Entities;
-using Recipes.DataAccess.Entities.Relationships;
-using Recipes.DataAccess.Filtering;
-using Recipes.DataAccess.Repositories;
+using Recipes.Business.Services.Interfaces;
 using Recipes.Public;
 
 namespace Recipes.API.Controllers;
 
 [ApiController]
 [Route("recipes")]
-public class RecipesController : ControllerBase
+public class RecipesController(IRecipesService recipeService) : ControllerBase
 {
-    private readonly IRecipesRepository _recipesRepository;
-    private readonly IGenericRepository<TagRecipeEntity> _tagRecipeRepository;
-    private readonly IGenericRepository<TagEntity> _tagRepository;
-    
-    public RecipesController(
-        IRecipesRepository recipeRepository,
-        IGenericRepository<TagRecipeEntity> tagRecipeRepository,
-        IGenericRepository<TagEntity> tagRepository)
-    {
-        _tagRecipeRepository = tagRecipeRepository;
-        _recipesRepository = recipeRepository;
-        _tagRepository = tagRepository;
-    }
-
     [HttpGet]
     [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
```

Comment:
Can it really output 401? We do not require a user to authenticate to fetch data.

Reply:
same with get by id

Reply:
removed